### PR TITLE
Addition of computer systems researchers from University of Amsterdam

### PR DESCRIPTION
**Pull request by adpimentel**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-a.csv [hunk 1]
```diff
@@ -1220,7 +1221,9 @@ Ana Maria Dinis Moreira,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl
 Ana Maria Paiva,Universidade de Lisboa,http://gaips.inesc-id.pt/~apaiva/Ana_Paiva_Site_2/Home.html,dTG0kWAAAAAJ
 Ana Matran-Fernandez,University of Essex,https://www.essex.ac.uk/people/matra32701/ana-matran-fernandez,_g01zM4AAAAJ
 Ana L. Milanova,Rensselaer Polytechnic Institute,http://www.cs.rpi.edu/~milanova,PsGxHG0AAAAJ
+Ana Lucia Varbanescu,University of Amsterdam,https://www.uva.nl/en/profile/v/a/a.l.varbanescu/a.l.varbanescu.html
 Ana Moreira 0001,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/12,uB4MBw4AAAAJ
+Ana Oprescu,University of Amsterdam,https://cci-research.nl/author/ana-oprescu/
```
Comment (emeryberger) [inline csrankings-a.csv:1226]: This faculty member has two aliases in DBLP, and both must be entered. See https://dblp.org/pid/20/1283.html.

Comment (emeryberger) [reply csrankings-a.csv:1226]: Also, there is no Google Scholar ID here.

File: csrankings-a.csv [hunk 2]
```diff
@@ -195,6 +195,7 @@ Adam A. Porter,University of Maryland - College Park,https://www.cs.umd.edu/user
 Adam Bargteil,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~adamb,fArWdfAAAAAJ
 Adam Bates 0001,Univ. of Illinois at Urbana-Champaign,http://adambates.org,1dVglBkAAAAJ
 Adam Belay,Massachusetts Institute of Technology,https://engineering.mit.edu/faculty-research/year-2017,AKQec5wAAAAJ
+Adam Belloum,University of Amsterdam,https://www.uva.nl/profiel/b/e/a.s.z.belloum/a.s.z.belloum.html
```
Comment (emeryberger) [inline csrankings-a.csv:198]: Missing Google Scholar ID.

File: csrankings-a.csv [hunk 3]
```diff
@@ -1573,6 +1576,7 @@ Andy Cobley,University of Dundee,https://www.computing.dundee.ac.uk/about/staff/
 Andy Cockburn,University of Canterbury,http://www.cosc.canterbury.ac.nz/andrew.cockburn,HzupDnMAAAAJ
 Andy Crabtree,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/andy.crabtree,LX_P5GYAAAAJ
 Andy D. Perkins,Mississippi State University,http://web.cse.msstate.edu/%7Eperkins,NOSCHOLARPAGE
+Andy D. Pimentel,University of Amsterdam,https://staff.fnwi.uva.nl/a.d.pimentel/
```
Comment (emeryberger) [inline csrankings-a.csv:1579]: Third "no Google Scholar ID". Closing. Please fix all and check all DBLP names for aliases, and add them, then re-open.

**Review by emeryberger — CHANGES_REQUESTED** [review]
See comments.

**Comment by emeryberger** [comment]
Re-open when comments have been addressed; please confirm that you have updated all entries.

**Comment by adpimentel** [comment]
Hi,

All entries have been updated/corrected.

BR,
    Andy Pimentel

On 27 Nov 2022, at 22:38, Emery Berger ***@***.***> wrote:



Re-open when comments have been addressed; please confirm that you have updated all entries.

—
Reply to this email directly, view it on GitHub<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Fpull%2F5188%23issuecomment-1328345667&data=05%7C01%7Ca.d.pimentel%40uva.nl%7Cb9f1969132ab4016950608dad0bfb1fd%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051818983444658%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8DEQJSBz4JarDl9PSGafd2HKRb1LoQSRZK4y6hft62A%3D&reserved=0>, or unsubscribe<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUSUKDEQ6QQRDLZ3IPY4OR3WKPIEPANCNFSM6AAAAAARKEG2TM&data=05%7C01%7Ca.d.pimentel%40uva.nl%7Cb9f1969132ab4016950608dad0bfb1fd%7Ca0f1cacd618c4403b94576fb3d6874e5%7C0%7C0%7C638051818983444658%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=KlvlVG3yvPMfKLpcUvd%2F3h4D%2BPuxvQiX2ifQ9muc1ZY%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

----
Andy D. Pimentel
Professor of Computer Science
Parallel Computing Systems group
University of Amsterdam
www.pcs-research.nl

**Pull request closed without merging**